contents/uix/accessibility.md

```diff
@@ -0,0 +1,141 @@
+
+### Visual
+Users with visual impairments rely on assistive technologies such as a magnifier or a screen reader.
+
+  * Use more visual indicators to convey a message.
+  
+  Let us look at Facebook sign up page. Suppose I want to sign up for a new account, but I have not put in all the necessary information. So, it is supposed to tell me that my attempt to sign up failed.
+
+  This page below uses a red color border around the text box to show that the information needed is missing. As red usually signifies failure, this seems enough.
+  ![Facebook sign up page](accessibility/fb-normal.png "Facebook sign up page")
+
+  However, to someone with color blindness, this is how the page looks like
+  ![Facebook sign up page to someone with achromatopsia](accessibility/fb-achromatopsia.png "Facebook sign up page to someone with achromatopsia")
+  _[Achromatopsia](https://en.wikipedia.org/wiki/Achromatopsia)_
+
+  ![Facebook sign up page to someone with deuteranopia](accessibility/fb-deuteranopia.png "Facebook sign up page to someone with deuteranopia")
+  _[Deuteranopia](https://en.wikipedia.org/wiki/Deuteranopia)_
+
+  It is unclear to color-blind users that the sign up has failed. To avoid confusion, we should not rely on colours alone to convey a message. Instead, we should use more visual indicators such as icons or an explanation box.
+  ![Sign up page with more visual indicators](accessibility/fb-more-indicators.png "Facebook sign up page with more visual indicators")
+
+  * Maintain good contrast
+  
+  Look at this page taken from Tech Crunch.
+  ![Tech Crunch page](accessibility/tc-normal.png "Tech Crunch page")
+
+  To users with good vision, the design may look minimalist and clean. However, to someone suffering from cataract, the page looks like this.
+
+  ![Blurry Tech Crunch page](accessibility/tc-blurred.png "Tech Crunch page to someone with cataract")
+  It gets difficult to read the news snippets because of the poor color contrast (grey on white). On the other hand, the black colored texts are still legible. Thus, we should maintain good contrast ratio in our websites for ease of reading.
+
+  * Use `alt` attribute for images
+  
+  The `alt` attribute provides alternative information for an image that can be read out by a screen reader to describe an image.
+
+  ![Use of alt tag](accessibility/alt-tag.png "Alt tag")
+
+  
+### Auditory
+Hearing-impaired users do use assistive technologies such as a hearing aid or a cochlear implant, but these are not specific for accessing websites.
+
+For users with hearing impairment, we should provide text alternatives to audio content, such as:
+  * Text transcripts
+  * Captions
+
+  
+### Cognitive
+Cognitive disabilities range from mental illnesses to learning difficulties, difficulties in comprehension and concentration, etc. Some examples include ADHD (attention deficit hyperactivity disorder), and autism.
```

Comment:
better to write Attention Deficit Hyperactivity Disorder (ADHD) instead. and consider including a link?